sci-calculators/bc-gh/bc-gh-6.2.2.ebuild

```
@@ -0,0 +1,44 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="https://git.yzena.com/gavin/bc"
+SRC_URI="https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE="libedit readline"
+
+RDEPEND="
+	!readline? ( libedit? ( dev-libs/libedit:= ) )
+	readline? (
+		sys-libs/readline:=
+		sys-libs/ncurses:=
+	)"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/bc-${PV}"
+
+src_configure() {
+
+	if use readline ; then
+		myconf="-r"
+	elif use libedit ; then
+		myconf="-e"
+	else
+		myconf=""
+	fi
```

```suggestion
	local myconf=()
	if use readline ; then
		myconf+=( -r )
	elif use libedit ; then
		myconf+=( -e )
	fi
```

No implicit word splitting, please.